src/tests/option_round/state_transition/auction_end/options_minted_tests.cairo

```diff
@@ -180,27 +175,18 @@ fn test_option_balance_per_bidder_after_auction_1() {
     // Make bids
     let option_bidders = option_bidders_get(4);
 
-    let bid_option_count_user_1: u256 = (params.total_options_available / 3);
     let bid_price_per_unit_user_1: u256 = params.reserve_price + 1;
-    let bid_amount_user_1: u256 = bid_option_count_user_1 * bid_price_per_unit_user_1;
-
-    let bid_option_count_user_2: u256 = (params.total_options_available / 3);
     let bid_price_per_unit_user_2: u256 = params.reserve_price + 2;
-    let bid_amount_user_2: u256 = bid_option_count_user_2 * bid_price_per_unit_user_2;
-
-    let bid_option_count_user_3: u256 = (params.total_options_available / 3);
     let bid_price_per_unit_user_3: u256 = params.reserve_price + 3;
-    let bid_amount_user_3: u256 = bid_option_count_user_3 * bid_price_per_unit_user_3;
-
-    let bid_option_count_user_4: u256 = (params.total_options_available / 3);
     let bid_price_per_unit_user_4: u256 = params.reserve_price + 4;
-    let bid_amount_user_4: u256 = bid_option_count_user_4 * bid_price_per_unit_user_4;
+
+    let bid_amounts = create_default_option_amount_array(4, params.total_options_available / 3);
```

Comment:
create_linear_array for bid amounts, and can use gradient array starting at reserve price and stepping 1